public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/CreateNew/CreateNew.js

```diff
@@ -0,0 +1,474 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState } from 'react';
+import { 
+  EuiTitle, 
+  EuiSpacer, 
+  EuiIcon, 
+  EuiText, 
+  EuiSwitch, 
+  EuiLoadingSpinner,
+  EuiPanel,
+  EuiAccordion,
+  EuiFormRow,
+  EuiFieldText,
+  EuiCheckbox,
+  EuiSelect,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiFieldNumber,
+  EuiCallOut
+ } from '@elastic/eui';
+import { EmbeddablePanel } from '../../../../../../../src/plugins/embeddable/public';
+import './styles.scss';
+import EnhancedAccordion from '../../EnhancedAccordion';
+
+function CreateNew({ embeddable, closeFlyout, core, services, index }) {
+  const [isShowVis, setIsShowVis] = useState(false);
+  const title = embeddable.getTitle();
+  const history = {
+    location: { pathname: '/create-detector', search: '', hash: '', state: undefined },
+    push: (value) => console.log('pushed', value),
+    goBack: closeFlyout,
+  };
+  const createMonitorProps = {
+    ...history,
+    history,
+    httpClient: core.http,
+    // This is not expected to be used
+    setFlyout: () => null,
+    notifications: core.notifications,
+    isDarkMode: core.isDarkMode,
+    notificationService: services.notificationService,
+    edit: false,
+    updateMonitor: () => null,
+    staticContext: undefined,
+    isMinimal: true,
+    defaultName: `${title} anomaly detector 1`,
```

Comment:
detector name can't have spaces. Also, we may want to think how to handle the case of multiple detectors being created from the same vis title. A different vis (or the same vis) may have the same title, which would produce the same default